tests/storage/logs/write_2019-07-10_11:48:52.log

```diff
@@ -0,0 +1,13 @@
+[INFO] 1562739532.9697| write Tests starting...
```

Comment:
why do we have a logs folder? Any example or something? Because if they are examples, maybe they should live in a docs folder.

Reply:
Sorry it was added by mistake, it's an example though.